src/types/paragraph.d.ts

```diff
@@ -143,6 +181,19 @@ export interface ParagraphServiceLocationAddress extends DrupalParagraph {
   field_use_facility_address: boolean
   field_wing_floor_or_room_number: string
 }
+
+export interface ParagraphStaffProfile extends DrupalParagraph {
+  field_first_name: fieldFirstName
+  field_last_name: fieldLastName
+  field_email_address: fieldEmailAddress
+  field_phone_number: fieldPhoneNumber
+  field_suffix: fieldSuffix
+  field_description: fieldDescription
+  field_complete_biography_create: fieldCompleteBiographyCreate
+  field_media: fieldMedia
+  field_entity: fieldEntity
+}
```

Comment:
Doesn't need to be this ticket but at some point we'll need descriptions so type doc picks everything up. Maybe we should create a task?